README.md

```diff
@@ -7,7 +7,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 ![Rails error](http://i.imgur.com/9NR7XNT.png)  
 
 ```text
-Your answer...
+I'm missing and index.html.erb template file.
```

Comment:
:+1:

I recommend you indicate the path when that is an important fact.  Is index.html.er in "app/views/users", "app/views/artists", "app/models"